.docker/readme.md

```diff
@@ -0,0 +1,55 @@
+If you'd like to only setup WordCamp.org and would like to use Docker, you can use inbuilt docker provisioning. Please follow these steps
```

Comment:
```suggestion
Follow these steps to setup WordCamp.org using [Docker](https://www.docker.com/).
```

`SHOULD` - Now that this is in a separate file, the if you'd like too... seems like an artifact leftover from when this sentence was next to one referring to the Meta Environment. If they're loading this file, we can assume they just want the Docker process.

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/113/commits/41867e7168934f7be3093a8ade78a9c1d3604520